zenodo/base/format_templates/DataCite.xsl

```diff
@@ -71,6 +71,20 @@ exclude-result-prefixes="marc fn dc invenio">
                 <creatorName>
                     <xsl:value-of select="subfield[@code='a']"/>
                 </creatorName>
+                <xsl:for-each select="subfield[@code='0']">
+                    <xsl:if test="substring(., 2, 5) = 'orcid'">
+                        <nameIdentifier schemeURI="http://orcid.org" nameIdentifierScheme="ORCID">
+                            <!-- parse only id from (orcid)xxxx-xxxx-xxxx -->
+                            http://orcid.org/<xsl:value-of select="substring(., 8)"/>
+                        </nameIdentifier>
+                    </xsl:if>
+                    <xsl:if test="substring(., 2, 3) = 'gnd'">
```

Comment:
DataCite prevents having multiple. Will be fixed in future versions of the schema but we need to remove it for now.